internal/context/context.go

```diff
@@ -154,6 +158,12 @@ func InitSmfContext(config *factory.Config) {
 			smfContext.CPNodeID.NodeIdType = pfcpType.NodeIdTypeIpv6Address
 			smfContext.CPNodeID.IP = addr.IP
 		}
+
+		if pfcp.AlertInterval == 0 {
```

Comment:
if AlertInterval is 0, smf should not launch the retry mechanism.

Reply:
`AssociationSetupFailedAlertInterval` represents the interval at which error messages are output, not retry association.
Our system monitors error messages and notifies operators, this value is used to reduce the frequency of notifications when the same error message is output consecutively.

Reply:
ok, but it is a little bit confusing that no alert trigger in code and only retry to set up association.